.pre-commit-config.yaml

```diff
@@ -199,7 +199,7 @@ repos:
     hooks:
       - id: isort
         name: Run isort to sort imports
-        types: [python]
+        files: \.py$
```

Comment:
Types: Python is not perfect. For some reason the types selection is sometimes wrong. We've learned that using file extensions is much more stable and predictable.